fudge-ratio/target/fudge_ratio-1/WEB-INF/appengine-web.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
THIS IS NOT YOUR PROBLEM:
But when Giovanna created the repo she created a .git-ignore file, so we wont end up with files in the target folder, but I believe something went wrong, I do not thing there is much we can do beside delete them and check what is going on with the gitignore in another commit

Reply:
I was not aware that a target folder was created. Good to know.